app/src/main/java/org/dpppt/android/app/network/InfoStatusNetwork.java

```diff
@@ -0,0 +1,40 @@
+package org.dpppt.android.app.network;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import org.dpppt.android.app.util.DebugUtils;
+
+public class InfoStatusNetwork {
+
+    public static final String UPDATE_NETWORK_STATUS = "org.dpppt.android.network.UPDATE_STATUS";
+
+
+    public static boolean isConnect(Context context)
+    {
+        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        assert connectivityManager != null;
+
+        NetworkInfo activeNetwork = connectivityManager.getActiveNetworkInfo();
+        return activeNetwork != null;
+    }
+
+    public static IntentFilter getUpdateIntentFilter() {
+        return new IntentFilter(UPDATE_NETWORK_STATUS);
+    }
+
+    public static void sendUpdateBroadcast(Context context) {
```

Comment:
Helper method to notify the change of the network, maybe be put in other class but for now, I think that this is a good choice